Makefile

```diff
@@ -155,8 +137,18 @@ ubuntu_udev_uninstall:
 	@echo "====================================================="
 	rm -f $(DESTDIR)/lib/udev/rules.d/99-razer.rules $(DESTDIR)/lib/udev/razer_mount
 
+appstream_install:
+	@echo -e "\n::\033[34m Installing OpenRazer AppStream metadata\033[0m"
+	@echo "====================================================="
+	install -m 644 -v -D install_files/appstream/io.github.openrazer.openrazer.metainfo.xml $(DESTDIR)/usr/share/metainfo/io.github.openrazer.openrazer.metainfo.xml
```

Comment:
`io.github.openrazer.openrazer` sounds weird

Reply:
I agree but it follows the specification. Look the example for 7-zip in the [AppStream specification](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#sect-Metadata-GenericComponent), it looks even weird.